newbank/server/NewBank.java

```diff
@@ -1,55 +1,77 @@
 package newbank.server;
 
 import java.util.HashMap;
+import java.util.ArrayList;
 
 public class NewBank {
-  
-  private static final NewBank bank = new NewBank();
-  private HashMap<String,Customer> customers;
-  
-  private NewBank() {
-    customers = new HashMap<>();
-    addTestData();
-  }
-  
-  private void addTestData() {
-    Customer bhagy = new Customer();
-    bhagy.addAccount(new Account("Main", 1000.0));
-    customers.put("Bhagy", bhagy);
-    
-    Customer christina = new Customer();
-    christina.addAccount(new Account("Savings", 1500.0));
-    customers.put("Christina", christina);
-    
-    Customer john = new Customer();
-    john.addAccount(new Account("Checking", 250.0));
-    customers.put("John", john);
-  }
-  
-  public static NewBank getBank() {
-    return bank;
-  }
-  
-  public synchronized CustomerID checkLogInDetails(String userName, String password) {
-    if(customers.containsKey(userName)) {
-      return new CustomerID(userName);
-    }
-    return null;
-  }
+	
+	private static final NewBank bank = new NewBank();
+	private HashMap<String,Customer> customers;
+	private ArrayList<String> commands = new ArrayList<String>();
+	
+	private NewBank() {
+		customers = new HashMap<>();
+		addTestData();
+		addCommands(commands);
+	}
+	
+	private void addTestData() {
+		Customer bhagy = new Customer();
+		bhagy.addAccount(new Account("Main", 1000.0));
+		customers.put("Bhagy", bhagy);
+		
+		Customer christina = new Customer();
+		christina.addAccount(new Account("Savings", 1500.0));
+		customers.put("Christina", christina);
+		
+		Customer john = new Customer();
+		john.addAccount(new Account("Checking", 250.0));
+		customers.put("John", john);
+	}
+	
+    private void addCommands(ArrayList<String> commands) {
+    	// user command and description
+		commands.add("SHOWMYACCOUNTS -> Lists all of your active accounts.");
+		commands.add("NEWACCOUNT -> Creates a new account of the specified type e.g. Current, Savings");
+		commands.add("LOGOUT -> Ends the current banking session and logs you out of NewBank.");

```

Comment:
Hi Heather,
Can you add the syntax for the NEWACCOUNT command i.e. NEWACCOUNT <name of account> e.g. "NEWACCOUNT savings"?

Reply:
Added syntax to command.